GW2EIParser/Settings/SettingsForm.Designer.cs

```diff
@@ -449,10 +453,39 @@ private void InitializeComponent()
             this.ChkHtmlExternalScripts.Text = "External Scripts";
             this.TlpSettings.SetToolTip(this.ChkHtmlExternalScripts, "Writes static css and js scripts in own files, which are shared between all logs." +
         " Log file size decreases, but the script files have to be kept along with the ht" +
-        "ml.");
+        "ml if you dont set a CDN Path");
             this.ChkHtmlExternalScripts.UseVisualStyleBackColor = true;
             this.ChkHtmlExternalScripts.CheckedChanged += new System.EventHandler(this.ChkHtmlExternalScriptsCheckedChanged);
             // 
+            // ChkDetailledWvW
+            // 
+            this.ChkDetailledWvW.AutoSize = true;
+            this.ChkDetailledWvW.Location = new System.Drawing.Point(6, 88);
+            this.ChkDetailledWvW.Name = "ChkDetailledWvW";
+            this.ChkDetailledWvW.Size = new System.Drawing.Size(136, 17);
+            this.ChkDetailledWvW.TabIndex = 41;
+            this.ChkDetailledWvW.Text = "Detailled WvW Parsing";
+            this.TlpSettings.SetToolTip(this.ChkDetailledWvW, "Keep default value if unsure. Enabling this will make parsing significantly slowe" +
+        "r and the generated files bigger");
+            this.ChkDetailledWvW.UseVisualStyleBackColor = true;
+            this.ChkDetailledWvW.CheckedChanged += new System.EventHandler(this.ChkDetailledWvWCheckedChange);
+            // 
+            // TxtHtmlExternalScriptsCdn
+            // 
+            this.TxtHtmlExternalScriptsCdn.Location = new System.Drawing.Point(44, 54);
+            this.TxtHtmlExternalScriptsCdn.Name = "TxtHtmlExternalScriptsCdn";
+            this.TxtHtmlExternalScriptsCdn.Size = new System.Drawing.Size(157, 20);
+            this.TxtHtmlExternalScriptsCdn.TabIndex = 57;
```

Comment:
A tooltip here to indicate this has priority over ScriptsPath and also warn about CORS related errors would be nice